src/screens/DiscoveryScreenV2/elements/DiscoveryTab.js

```diff
@@ -47,38 +67,41 @@ const styles = StyleSheet.create({
     height: 48,
     backgroundColor: colors.white
   },
-  tabItem: (index) => ({
-    flex: 1,
-    width: width / 4,
+  tabItem: (numTabs) => ({
+    width: numTabs === 4 ? undefined : windowWidth / numTabs,
     justifyContent: 'center',
+    alignItems: 'center',
     height: '100%',
-    paddingLeft: index !== 1 ? 20 : 0
+    paddingHorizontal: numTabs === 4 ? 20 : undefined
   }),
   tabItemContainer: {
-    alignSelf: 'flex-start'
+    alignSelf: 'center'
   },
   tabItemText: {
     color: colors.alto,
-    fontFamily: fonts.inter[500],
+    fontWeight: '500',
     fontSize: normalizeFontSize(12.5),
-    // lineHeight: 16.94,
     paddingVertical: 10,
-    textAlign: 'left'
+    textAlign: 'center'
   },
   tabItemTextFocus: {
     color: colors.black,
-    fontFamily: fonts.inter[500],
+    fontWeight: '500',
     fontSize: normalizeFontSize(12.5),
     lineHeight: 16.94,
-    textAlign: 'left',
+    textAlign: 'center',
     paddingVertical: 10
   },
   underlineFocus: {
     borderBottomColor: colors.bondi_blue,
-    borderBottomWidth: 2,
-    top: 0,
-    position: 'relative'
+    borderBottomWidth: 2
   }
 });
 
+DiscoveryTab.propTypes = {
+  onChangeScreen: PropTypes.func,
+  selectedScreen: PropTypes.number,
+  tabs: PropTypes.object
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

PropTypes have been added to the component. This is a good practice for type checking in React.
```diff
+ DiscoveryTab.propTypes = {
+   onChangeScreen: PropTypes.func,
+   selectedScreen: PropTypes.number,
+   tabs: PropTypes.object
+ };
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->